app/models/product.rb

```diff
@@ -1,4 +1,5 @@
 # frozen_string_literal: true
 
 class Product < ApplicationRecord
+  validates :product_details, length: { minimum: 1 }
```

Comment:
You can also add the validation for the 5 `product_details`:
`validates :product_details, inclusion: { in: %w(small medium large light heavy) }`